src/main/java/org/lsc/plugins/connectors/executable/AbstractExecutableLdifService.java

```diff
@@ -130,28 +132,26 @@ private int execute(String[] runtime, String[] env, String input, StringBuffer d
             outputStream.flush();
             outputStream.close();
 
+            BufferedReader reader = new BufferedReader(new InputStreamReader(p.getInputStream()));
+            String line;
+            while ((line = reader.readLine()) != null) {
+                datas.append(line + System.getProperty("line.separator"));
+            }

```

Comment:
why not simply using IOUtils.toString?

Reply:
good point, I've changed it to use IOUtils.toString